### DISCUSSION TITLE
txt to markdown, preserving txt formatting

### DISCUSSION THREAD

**Discussion by sixhex** [gsantner/markor#2474] [Q&A] [answered]
hi!

i would like to be able to convert txt based copy/paste (or txt files) to markdown preserving the end result formatting as much as possible.

example:

------
txt:

He’s got a road map of Jupiter
A radar fix on the stars
All along the highway
She’s got a liquid-crystal compass
A picture book of the rivers
Under the Sahara

They travel in the time of the prophets
On a desert highway straight to the heart of the sun
Like lovers and heroes, and the restless part of everyone
We’re only at home when we’re on the run
On the run

------

becomes 

He’s got a road map of Jupiter A radar fix on the stars All along the highway She’s got a liquid-crystal compass A picture book of the rivers Under the Sahara

They travel in the time of the prophets On a desert highway straight to the heart of the sun Like lovers and heroes, and the restless part of everyone We’re only at home when we’re on the run On the run

------

when i attempt to convert to markdown by edit (pen button)->hamburger menu->file settings->format->markdown (was originally plain text).

i do this a lot while studying and writing lyrics, and would prefer that i can convert to markdown while preserving line breaks without having to double linebreaks manually.

thanks!

--=

i am having a rough morning and i *did* spend a good few minutes searching permutations of [text/txt/convert/markdown/crlf/line break/preserve formatting]. i apologize if this is an existing feature i'm too thick/daft to find this morning.

thank you!

**Comment by gsantner** [comment, marked as answer]
You can also use Markor to view and edit plaintext. Just because Markdown is kind of natural choice are not forced to use it (and it's way how paragraphs work in Mardown.

You can switch at the top bar to Format - Plaintext, or sekect Plaintext at the New File dialog.

Gregor
